tests/system/marathon_pods_tests.py

```diff
@@ -545,7 +545,12 @@ def wait_for_status_network_info():
     task_id1 = tasks[0]['id']
     task_id2 = tasks[1]['id']
 
-    shakedown.kill_process_on_host(host, '[h]ttp.server')
+    @retrying.retry(wait_fixed=1000, stop_max_attempt_number=30, retry_on_exception=common.ignore_exception)
+    def kill_task(host, pattern):
+        killed = common.kill_process_on_host(host, pattern)
+        assert killed, "no task got killed on {} for pattern {}".format(host, pattern)
+
+    kill_task(host, '[h]ttp.server')
```

Comment:
`'[h]ttp.server' ` is the same as `'http.server' `, right? What is the intention with the square brackets here?

Also, `.` will match any character? Should we escape that?